spec/snapshots/Kitchen_Directions_BakeLearningObjectives_when_only_li_need_to_be_counted_in_appendix_works.snap

```diff
@@ -0,0 +1,14 @@
+<div data-type="chapter">
```

Comment:
this test data output doesn't seem to show the learning objective baking with numbers ... am i reading this right?

Reply:
Yes, it shouldn't look like that. I will look into it.